apps/fortuna/Cargo.toml

```diff
@@ -19,7 +19,7 @@ ethabi = "18.0.0"
 ethers = { version = "2.0.14", features = ["ws"] }
 futures = { version = "0.3.28" }
 hex = "0.4.3"
-prometheus-client = { version = "0.21.2" }
+prometheus-client = { version = "0.23.1" }
```

Comment:
This upgrade is required because in 0.21.2 there was no implementation of the `EncodeLabelSet` trait for `bool`, which I need